test/csuite/random_directio/main.c

```diff
@@ -342,13 +341,9 @@ gen_table_name(char *buf, size_t buf_size, uint64_t id, uint32_t threadid)
  *     Generate a second table name used for the schema test.
  */
 static void
-gen_table2_name(char *buf, size_t buf_size, uint64_t id, uint32_t threadid, uint32_t flags)
+gen_table2_name(char *buf, size_t buf_size, uint64_t id, uint32_t threadid)
 {
-    if (!LF_ISSET(SCHEMA_RENAME))
-        /* table is not renamed, so use original table name */
-        gen_table_name(buf, buf_size, id, threadid);
-    else
-        testutil_snprintf(buf, buf_size, "table:B%" PRIu64 "-%" PRIu32, id, threadid);
+    testutil_snprintf(buf, buf_size, "table:B%" PRIu64 "-%" PRIu32, id, threadid);
```

Comment:
Question: Is this right? The not rename case is deleted and the rename case is kept.